camunda-formio-tasklist-vue/src/components/TaskList.vue

```diff
@@ -925,6 +933,12 @@ export default class Tasklist extends Mixins(TaskListMixin) {
   }
 
   async mounted () {
+    const userDetails = JSON.parse(localStorage.getItem('UserDetails') ?? "") as UserDetailFromToken;
+    this.isUserAllowed = isAllowedUser(userDetails, this.formIOReviewer);
+    if (!this.isUserAllowed){
```

Comment:
Is this console statement required? We can try to avoid console statements maximum for codebase